api/v1alpha1/zz_generated.workflow.chaosmesh.go

```diff
@@ -13,25 +13,27 @@
 
 package v1alpha1
 
+
 import (
 	"fmt"
-
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+
 const (
-	TypeAwsChaos     TemplateType = "AwsChaos"
-	TypeDNSChaos     TemplateType = "DNSChaos"
-	TypeGcpChaos     TemplateType = "GcpChaos"
-	TypeHTTPChaos    TemplateType = "HTTPChaos"
-	TypeIoChaos      TemplateType = "IoChaos"
-	TypeJVMChaos     TemplateType = "JVMChaos"
-	TypeKernelChaos  TemplateType = "KernelChaos"
+	TypeAwsChaos TemplateType = "AwsChaos"
```

Comment:
I notice that the indent frequently modified. Maybe you should `make check` or something before commit this file?

Reply:
The `make check` command will fail currently, and I have executed `make fmt` command. Moreover, I think we should ignore generated file in any format tasks.

Reply:
Maybe we should change the Makefile.

Reply:
Ignored